doc/managing_wallet.md

```diff
@@ -0,0 +1,115 @@
+# Managing The Wallet
+
+## 1. Backing Up and Restoring The Wallet
+
+### 1.1 Creating The Wallet
+
+Since version 0.21, Bitcoin Core no longer has a default wallet.
+Wallets can be created with the RPC command `createwallet`. More information about this command can be found running `bitcoin-cli help createwallet`.
+
+The following command, for example, creates a descriptor wallet:
+
+`$ bitcoin-cli -named createwallet wallet_name="wallet01" descriptors=true`
+
+If the node is running on testnet network, `-testnet` parameter should be added. Or `-signet` if on signet.
+
+`$ bitcoin-cli -testnet -named createwallet wallet_name="wallet01" descriptors=true`
+
+The `descriptors` parameter can be omitted if the intention is to create a legacy wallet.
+
+For now, the default type is the legacy wallet, but that should change in the near future.
+
+By default, wallets are created in the `~/.bitcoin/wallets/wallet_name` folder. If the node is running on testnet, the wallets are created in `~/.bitcoin/testnet3/wallets/wallet_name` or if signet, `~/.bitcoin/signet/wallets/wallet_name`.
+
+### 1.2 Encrypting The Wallet
+
+The `wallet.dat` file is not encrypted by default and is, therefore, vulnerable if an attacker gains access to the device where the wallet or the backups are stored.
+
+The wallet must be encrypted with the following command:
+
+`$ bitcoin-cli -rpcwallet="wallet-01" encryptwallet "passphrase"`
+
+The `encryptwallet` command is used only when the wallet has not been encrypted yet. Otherwise, the `walletpassphrasechange` command should be used.
+
+`$ bitcoin-cli -rpcwallet="wallet-01" walletpassphrasechange "oldpassphrase" "newpassphrase"`
+
+The argument passed to `-rpcwallet` is the name of the wallet to be encrypted.
+
+The term "encrypt the wallet" used here is not very accurate. This command encrypts only the private key. All other wallet information, such as transactions, is still visible.
+
+Note that if the passphrase is lost, all the coins in the wallet will also be lost forever.
+
+### 1.3 Backing Up The Wallet
+
+Wallets files are portable and can be copied. For safety, one should store backup copies on offline devices only.
```

Comment:
Not directly. The `backupwallet` RPC or "Backup Wallet" GUI menu item must be used to ensure the file is in a safe state when the copy is made.

Reply:
Comment added.